peru/resources/plugins/git/git_plugin.py

```diff
@@ -167,10 +163,33 @@ def plugin_reup(url, reup):
         print('rev:', output.strip(), file=out_file)
 
 
+def git_default_branch(url) -> str:
+    """
+    With a list of common default branches names, it checks against
+    the available branch list if this possible default branch exists.
+    By default git used to take master as default branch, but newer versions
+    tend to use main, even trunk or development can be found.
```

Comment:
We could update this comment, since we've shortened the list. But I can also do that myself after landing, no worries.

Reply:
I do it, I am also going to simplify the function a little bit.